From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Sat, 19 Dec 2020 16:44:16 +0100
Subject: ANDROID: GKI: remove CONFIG_IOMMU_LIMIT_IOVA_ALIGNMENT=y

Commit 9a76e701d5cf ("Revert "FROMLIST: iommu/iova: Support limiting
IOVA alignment"") dropped the CONFIG_IOMMU_LIMIT_IOVA_ALIGNMENT config
option, so also remove it from the arm64 gki_defconfig file so that
everything continues to build properly.

Fixes: 9a76e701d5cf ("Revert "FROMLIST: iommu/iova: Support limiting IOVA alignment"")
Bug: 148141615
Cc: Liam Mark <lmark@codeaurora.org>
Cc: Chris Goldsworthy <cgoldswo@codeaurora.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: I3c8b38d4d3771f15023133eb82c6d638988e9045
---
 arch/arm64/configs/gki_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -465,7 +465,6 @@ CONFIG_HWSPINLOCK=y
 CONFIG_SUN4I_TIMER=y
 # CONFIG_SUN50I_ERRATUM_UNKNOWN1 is not set
 CONFIG_MAILBOX=y
-CONFIG_IOMMU_LIMIT_IOVA_ALIGNMENT=y
 CONFIG_REMOTEPROC=y
 CONFIG_REMOTEPROC_CDEV=y
 CONFIG_RPMSG_CHAR=y
